Lesson_Materials/Lecture_04_Handling_Errors/index.html

```diff
@@ -95,25 +95,22 @@
 class add;
 
 int main() {
-  std::vector&lt;float&gt; dA{ 7, 5, 16, 8 }, dB{ 8, 16, 5, 7 }, dO{ 0, 0, 0, 0 };
-    queue gpuQueue(gpu_selector{});
-    buffer bufA{dA};
-    buffer bufB{dB};
-    buffer bufO{dO};
+  queue q();
 
-    gpuQueue.submit([&](handler &cgh) {
-      auto inA = accessor{bufA, cgh, read_only};
-      auto inB = accessor{bufB, cgh, read_only};
-      auto out = accessor{bufO, cgh, write_only};
+  /* Synchronous code */ 
 
-      cgh.single_task&lt;add&gt;(bufO.get_range(), [=](id&lt;1&gt; i) {
-        out[i] = inA[i] + inB[i];
-      });
-    }).wait();
+  q.submit([&](handler &cgh) {
+    /* Synchronous code */
+
+    cgh.single_task&lt;add&gt;([=](id&lt;1&gt; i) {
+      /* Asynchronous code */
+    });
+  }).wait();
 }
 						</code></pre>
 					</div>
 					<div class="bottom-bullets" data-markdown>
+					* Code on the device runs synchronously
```

Comment:
```suggestion
					* Code on the device runs asynchronously
```

probably should have actually read what it was saying first time around :laughing: